crystal_functions/file_readwrite.py

```diff
@@ -10,10 +10,10 @@ class Crystal_input:
 
     def __init__(self):
         # Initialise the object
-        
+
```

Comment:
Please ignore these blank lines - I accidentally launched auto PEP8 and modified the spaces and blank lines to meet the PEP8 standard. That doesn't interfere the functionalities.